sonar-ws/src/main/java/org/sonarqube/ws/client/project/CreateRequest.java

```diff
@@ -89,6 +102,12 @@ public Builder setBranch(@Nullable String branch) {
       return this;
     }
 
+    public Builder setVisibility(@Nullable String visibility) {
+      checkArgument(visibility == null || asList("private", "public").contains(visibility), "Unexpected visibility '" + visibility + "'");
```

Comment:
seriously, you are repeating the constants "private" and "public" everywhere, drop this duplication, it's going to be a maintenance nightmare